R/run.R

```diff
@@ -524,20 +524,10 @@ log_table_to_run <- function(name, value, description = "", run = NULL) {
   invisible(NULL)
 }
 
-#' Plot table of run details
-#' @description
-#' Plot table of run details in RStudio Viewer or browser.
-#' This table does not auto-refresh. To see current values,
-#' re-run the command or click the web view link to view more
-#' details in real time.
-#'
-#' If you are running this method from an RMarkdown file, the
-#' run details table will show up in the code chunk output
-#' instead of the Viewer.
+#' Generate table of run details
 #' @param run The `Run` object.
-#' @export
 #' @md
-view_run_details <- function(run) {
+create_run_details_plot <- function(run) {
```

Comment:
I agree with adding the leading dot. Not sure about renaming because that function doesn't necessarily plot anything. It returns a datatable object that is passed to the Shiny background job when in RStudio. When not in RStudio, it does plot that table. I think create is the better descriptor.

Reply:
ok then let's just add the leading dot.